src/utils/notifications.ts

```diff
@@ -278,7 +278,11 @@ export const sendMentionNotifications = async ({
   expenditureID,
   recipients,
 }: MentionNotificationVariables): Promise<void> => {
-  if (!recipients.length) {
+  const nonSelfRecipients = recipients.filter(
```

Comment:
@davecreaser maybe we could rename this to `recipientsExcludingCreator`? I think it's more suggestive 🤔